daemon/src/model.rs

```diff
@@ -390,7 +390,7 @@ impl From<Decimal> for Percent {
     }
 }
 
-#[derive(Debug, Clone, Serialize, Deserialize, PartialEq, sqlx::Type)]
+#[derive(Debug, Clone, Copy, Serialize, Deserialize, PartialEq, sqlx::Type)]
 pub enum TradingPair {
```

Comment:
do you need to implement `Clone` anymore here?

Reply:
You need `Clone` to derive `Copy`!

Reply:
ah, indeed!